db/databaseconnection.js

```diff
@@ -0,0 +1,29 @@
+const { Client } = require('pg')
+const client = new Client({
+  user: 'emmanuel',
```

Comment:
I think this is where the issue is, on my computer there is no emmanuel - I think you would need to have an env variable here perhaps that takes the current user? I'm not sure though